### PR TITLE
fix: typos with spell checker

### PR DESCRIPTION
**Pull request by azukiboy**
---

#### PR checklist

- [ ] Tests written/updated
- [ ] Changelog entry added in `.changelog` (we use [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [ ] Updated relevant documentation (`docs/` or `spec/`) and code comments

1. fix all text typo with spell checker(mainly *.md)
2. go,proto and other file types are excluded to avoid possible problems with proper nouns
3. CI may be considered in the next step ref:https://github.com/codespell-project/codespell

### CODE REVIEW

File: spec/light-client/accountability/TendermintAccInv_004_draft.tla [hunk 1]
```diff
@@ -273,7 +273,7 @@ AllNoEquivocationByCorrect ==
 \* @type: (Set({ src: $process, a })) => Set($process);
 Senders(M) == { m.src: m \in M }
 
-\* The final piece by Josef Widder:
+\* The final piece by Josef Wider:
```
Comment (melekes) [inline spec/light-client/accountability/TendermintAccInv_004_draft.tla:276]:
```suggestion
\* The final piece by Josef Widder:
```

File: spec/consensus/proposer-based-timestamp/tla/TendermintPBT_001_draft.tla [hunk 2]
```diff
@@ -8,7 +8,7 @@
  * Version 1. A preliminary specification.
 
  Zarko Milosevic, Igor Konnov, Informal Systems, 2019-2020.
- Ilina Stoilkovska, Josef Widder, Informal Systems, 2021.
+ Ilina Stoilkovska, Josef Wider, Informal Systems, 2021.
```
Comment (melekes) [inline spec/consensus/proposer-based-timestamp/tla/TendermintPBT_001_draft.tla:11]:
```suggestion
 Ilina Stoilkovska, Josef Widder, Informal Systems, 2021.
```

File: spec/consensus/consensus-paper/lit.bib [hunk 3]
```diff
@@ -1171,15 +1171,15 @@ @article{VL96:ic
...
+  author =	 {Josef Wider and Günther Gridling and Bettina Weiss
                   and Jean-Paul Blanquart},
   year =	 {2007},
   booktitle =	 DSN07,
   publisher =	 IEEE
 }
 
 @inproceedings{Wid03:disc,
-  author =	 {Josef Widder},
+  author =	 {Josef Wider},
```
Comment (melekes) [inline spec/consensus/consensus-paper/lit.bib:1182]:
```suggestion
  author =	 {Josef Widder},
```

**Comment by azukiboy** [comment]
@melekes  fix some name

File: docs/rfc/rfc-104-actor-model.md [hunk 4]
```diff
@@ -198,6 +198,6 @@ progressive refactoring of reactors/services over time.
 [Akka]: https://akka.io/
 [Proto Actor]: https://proto.actor/
 [Quasar]: http://docs.paralleluniverse.co/quasar/
-[CAF]: https://www.actor-framework.org/
+[CALF]: https://www.actor-framework.org/
```
Comment (hoangdv2429) [inline docs/rfc/rfc-104-actor-model.md:201]: revert this

File: docs/rfc/rfc-104-actor-model.md [hunk 5]
```diff
@@ -88,7 +88,7 @@ different programming languages include:
 - [Proto Actor] (Go/C\#/Java/Kotlin)
 - [Akka] (Java/Scala/C\#)
 - [Quasar] (Java/Kotlin)
-- [CAF] (C++)
+- [CALF] (C++)
```
Comment (hoangdv2429) [inline docs/rfc/rfc-104-actor-model.md:91]: please revert this as well

File: spec/consensus/consensus-paper/IEEEtran.bst [hunk 6]
```diff
@@ -242,7 +242,7 @@ FUNCTION {bbl.part}{ "pt." }
 FUNCTION {bbl.patent}{ "Patent" }
 FUNCTION {bbl.patentUS}{ "U.S." }
 FUNCTION {bbl.revision}{ "Rev." }
-FUNCTION {bbl.series}{ "ser." }
+FUNCTION {bbl.series}{ "set." }
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/IEEEtran.bst:245]: revert this

File: spec/consensus/consensus-paper/IEEEtran.cls [hunk 7]
```diff
@@ -1000,7 +1000,7 @@
 % \leftmargin, the spacing from the left margin of the main text to
 % the left of the main body of a list item is set by \list.
 % Hence this statement does nothing for lists.
-% But, quote and verse do use it for indention.
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/IEEEtran.cls:1003]: this is not a typo

File: spec/consensus/consensus-paper/IEEEtran.cls [hunk 8]
```diff
@@ -1053,7 +1053,7 @@
 \def\labelenumiii{\theenumiii)} \def\theenumiii{\roman{enumiii}}
 
 % IEEE uses A) not A.
-\def\labelenumiv{\theenumiv)}   \def\theenumiv{\Alph{enumiv}}
+\def\labelenumiv{\theenumiv)}   \def\theenumiv{\Alpha{enumiv}}
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/IEEEtran.cls:1056]: this's also not a typo

File: spec/consensus/consensus-paper/IEEEtran.cls [hunk 9]
```diff
@@ -1077,7 +1077,7 @@
 % which ITEMIZED list labels are indented to the right
 % For The Journal of Lightwave Technology (JLT) and The Journal
 % on Selected Areas in Communications (JSAC), they tend to use
-% an indention equal to \parindent. For Transactions on Communications
+% an indentation equal to \parindent. For Transactions on Communications
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/IEEEtran.cls:1080]: revert this as well

File: spec/consensus/consensus-paper/IEEEtran.cls [hunk 10]
```diff
@@ -1095,13 +1095,13 @@
 
 % This controls the default amount the enumerated list labels
 % are indented to the right.
-% Normally, this is the same as the paragraph indention
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/IEEEtran.cls:1098]: revert this.

File: spec/consensus/consensus-paper/IEEEtran.cls [hunk 11]
```diff
@@ -1095,13 +1095,13 @@
...
+% Normally, this is the same as the paragraph indentation
 \newdimen\IEEEelabelindent
 \IEEEelabelindent \parindent
 
 % This controls the default amount the description list labels
 % are indented to the right.
-% Normally, this is the same as the paragraph indention
+% Normally, this is the same as the paragraph indentation
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/IEEEtran.cls:1104]: revert this.

File: spec/consensus/consensus-paper/IEEEtran.cls [hunk 12]
```diff
@@ -1121,7 +1121,7 @@
 % There may not be such a thing as correct "standard IEEE"
 % values. What IEEE actually does may depend on the specific
 % circumstances.
-% The first list level almost always has full indention.
+% The first list level almost always has full indentation.
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/IEEEtran.cls:1124]: revert this.

File: spec/consensus/consensus-paper/IEEEtran.cls [hunk 13]
```diff
@@ -1598,7 +1598,7 @@
 \else
 \def\thesection{\Roman{section}}                             % I
 % V1.7, \mbox prevents breaks around - 
-\def\thesubsection{\mbox{\thesection-\Alph{subsection}}}     % I-A
+\def\thesubsection{\mbox{\thesection-\Alpha{subsection}}}     % I-A
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/IEEEtran.cls:1601]: revert this.

File: spec/consensus/consensus-paper/IEEEtran.cls [hunk 14]
```diff
@@ -1629,7 +1629,7 @@
   \fi
 \else% not compsoc
   \def\thesectiondis{\thesection.}                   % I.
-  \def\thesubsectiondis{\Alph{subsection}.}          % B.
+  \def\thesubsectiondis{\Alpha{subsection}.}          % B.
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/IEEEtran.cls:1632]: revert this.

File: spec/consensus/consensus-paper/IEEEtran.cls [hunk 15]
```diff
@@ -3423,7 +3423,7 @@ between column types.}%
   \let\@makefnmark\relax% V1.7, must *really* kill footnotemark to remove all \textsuperscript spacing as well.
   \footnotesize%       equal spacing between thanks lines
   \footnotesep 0.7\baselineskip%see global setting of \footnotesep for more info
-  % V1.7 disable \thanks note indention for compsoc
+  % V1.7 disable \thanks note indentation for compsoc
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/IEEEtran.cls:3426]: revert this.

File: spec/consensus/consensus-paper/IEEEtran.cls [hunk 16]
```diff
@@ -4120,7 +4120,7 @@ between column types.}%
     \setcounter{paragraph}{0}%
     \gdef\thesection{A}%
     \gdef\thesectiondis{}% 
-    \gdef\thesubsection{\Alph{subsection}}%
+    \gdef\thesubsection{\Alpha{subsection}}%
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/IEEEtran.cls:4123]: revert this.

File: spec/consensus/consensus-paper/IEEEtran.cls [hunk 17]
```diff
@@ -4141,7 +4141,7 @@ between column types.}%
 % declare the individual appendices
 \def\appendices{\@IEEEsaveoriginalsectiononce\par
     % v1.6 keep hyperref's identifiers unique
-    \gdef\theHsection{Appendix.\Alph{section}}%
+    \gdef\theHsection{Appendix.\Alpha{section}}%
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/IEEEtran.cls:4144]: revert this.

File: spec/consensus/consensus-paper/IEEEtran.cls [hunk 18]
```diff
@@ -4154,10 +4154,10 @@ between column types.}%
...
     \gdef\@IEEEthmcounterinsection##1{A\arabic{##1}}
     \else%
-    \gdef\thesection{\Alph{section}}%
-    \gdef\thesectiondis{\Alph{section}}%
-    \@IEEEcompsocconfonly{\gdef\thesectiondis{\Alph{section}.}}%
-    \gdef\@IEEEthmcounterinsection##1{\Alph{##1}}
+    \gdef\thesection{\Alpha{section}}%
+    \gdef\thesectiondis{\Alpha{section}}%
+    \@IEEEcompsocconfonly{\gdef\thesectiondis{\Alpha{section}.}}%
+    \gdef\@IEEEthmcounterinsection##1{\Alpha{##1}}
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/IEEEtran.cls:4160]: revert this.

File: spec/consensus/consensus-paper/lit.bib [hunk 19]
```diff
@@ -363,7 +363,7 @@ @article{BDFG03:sigact
 @unpublished{BHR+06:note,
   author =	 "Martin Biely and Martin Hutle and Sergio Rajsbaum
                   and Ulrich Schmid and Corentin Travers and Josef
-                  Widder",
+                  Wider",
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/lit.bib:366]: revert this.

File: spec/consensus/consensus-paper/lit.bib [hunk 20]
```diff
@@ -1295,7 +1295,7 @@ @inproceedings{Li:srds07
  @article{Amir11:prime,
  author = {Amir, Yair and Coan, Brian and Kirsch, Jonathan and Lane, John},
  title = {Prime: Byzantine Replication under Attack},
- journal = {IEEE Trans. Dependable Secur. Comput.},
+ journal = {IEEE Trans. Dependable Secure. Comput.},
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/lit.bib:1298]: revert this.

File: spec/consensus/consensus-paper/lit.bib [hunk 21]
```diff
@@ -1376,7 +1376,7 @@ @inproceedings{Cach01:crypto
 @article{Moniz11:ritas,
  author = {Moniz, Henrique and Neves, Nuno Ferreria and Correia, Miguel and Verissimo, Paulo},
  title = {RITAS: Services for Randomized Intrusion Tolerance},
- journal = {IEEE Trans. Dependable Secur. Comput.},
+ journal = {IEEE Trans. Dependable Secure. Comput.},
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/lit.bib:1379]: revert this.

File: spec/consensus/consensus-paper/lit.bib [hunk 22]
```diff
@@ -1416,7 +1416,7 @@ @inproceedings{RMES:dsn07
 
 @article{Ben:jc92,
   author    = {Charles H. Bennett and
-               Fran\c{c}ois Bessette and
+               Fran\c{c}is Bessette and
```
Comment (hoangdv2429) [inline spec/consensus/consensus-paper/lit.bib:1419]: revert this.

File: spec/light-client/supervisor/supervisor_001_draft.tla [hunk 23]
```diff
@@ -67,5 +67,5 @@ InvEnoughPeers ==
 
 =============================================================================
 \* Modification History
-\* Last modified Sun Oct 18 11:48:45 CEST 2020 by widder
-\* Created Sun Oct 18 11:18:53 CEST 2020 by widder
+\* Last modified Sun Oct 18 11:48:45 CEST 2020 by wider
+\* Created Sun Oct 18 11:18:53 CEST 2020 by wider
```
Comment (hoangdv2429) [inline spec/light-client/supervisor/supervisor_001_draft.tla:71]: revert this.

**Review by hoangdv2429 — CHANGES_REQUESTED** [review]
overall everything is good. But please review changes before putting up a PR. Thanks!

**Comment by azukiboy** [comment]
@hoangdv2429  resolved all thks

**Pull request closed without merging**